pyodi/apps/train_config/train_config_evaluation.py

```diff
@@ -150,20 +155,25 @@ def train_config_evaluation(
         ```
     """
     if output is not None:
-        output = str(Path(output) / Path(ground_truth_file).stem)
         Path(output).mkdir(parents=True, exist_ok=True)
 
-    coco_ground_truth = load_ground_truth_file(ground_truth_file)
+    if isinstance(ground_truth_file, str):
+        coco_ground_truth = load_ground_truth_file(ground_truth_file)
 
-    df_images, df_annotations = coco_ground_truth_to_dfs(coco_ground_truth)
+        df_images, df_annotations = coco_ground_truth_to_dfs(coco_ground_truth)
 
-    df_annotations = join_annotations_with_image_sizes(df_annotations, df_images)
+        df_annotations = join_annotations_with_image_sizes(df_annotations, df_images)
 
-    df_annotations = filter_zero_area_bboxes(df_annotations)
+        df_annotations = filter_zero_area_bboxes(df_annotations)
 
-    df_annotations = scale_bbox_dimensions(df_annotations, input_size=input_size)
+        df_annotations = scale_bbox_dimensions(
+            df_annotations, input_size=input_size, keep_ratio=keep_ratio
+        )
+
+        df_annotations = get_scale_and_ratio(df_annotations, prefix="scaled")
```

Comment:
This lines are repeated in train-config generation and train-config evaluation...
We might move this into a new function but not sure about where and if it is really necessary